Sources/SPMBuildCore/BuildParameters/BuildParameters.swift

```diff
@@ -29,6 +29,9 @@ public struct BuildParameters: Encodable {
     /// The path to the data directory.
     public var dataPath: AbsolutePath
 
+    /// The path to the SDKROOT directory.
+    public var sdkPath: AbsolutePath?
```

Comment:
There's already `sdkRootPath` available on the `var toolchain: Toolchain` property below. Is this any different from that?

Reply:
Thank you @MaxDesiatov I addressed your comment.